internal/notifier/telegram.go

```diff
@@ -36,10 +36,29 @@ func (t *Telegram) Post(event events.Event) error {
 		emoji = "🚨"
 	}
 
-	message := fmt.Sprintf(`%s %s/%s/%s
+	message := fmt.Sprintf(`*%s %s/%s/%s*
+
 	%s`, emoji, strings.ToLower(event.InvolvedObject.Kind),
 		event.InvolvedObject.Name, event.InvolvedObject.Namespace, event.Message)
-	url := fmt.Sprintf("telegram://%s@telegram?channels=%s", t.Token, t.Channel)
-	err := shoutrrr.Send(url, message)
+	url := fmt.Sprintf("telegram://%s@telegram?channels=%s&parseMode=markDownv2", t.Token, t.Channel)
+	err := shoutrrr.Send(url, escapeString(message))
 	return err
 }
+
+func escapeString(str string) string {
```

Comment:
Is this something that shoutrrr requires or it's telegram? Can you please link to some docs around the escape logic?

Reply:
Telegram requires it when using Markdown. Docs: (https://core.telegram.org/bots/api#formatting-options)[https://core.telegram.org/bots/api#formatting-options]

Reply:
Ok please add a comment to this function including the link.